proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,332 @@
+  * This would replace the very buggy iOS-specific `shadow*` styles and Android `elevation` style, allowing developers to unify shadows across native and web platforms.
+* `fontVariant`
+  * Support space-separated string values to align with CSS.
+* `fontWeight`
+  * Support number values to align with React DOM / CSS.
+* `lineClamp`
+  * Support CSS clamping text to a set number of lines via style.
+  * This would be equivalent to using the `numberOfLines` prop on `<Text>` components.
+* `objectFit`
+  * Support CSS `objectFit` property for `<Image>`.
+  * This would be a subset (no "repeat" keyword) of functionality currently supported by the `resizeMode` prop and style for `<Image>`.
+  * `objectFit === 'contain'` => `resizeMode = 'contain'`
+  * `objectFit === 'cover'` => `resizeMode = 'cover'`
+  * `objectFit === 'fill'` => `resizeMode = 'stretch'`
+  * `objectFit === 'scale-down'` => `resizeMode = 'contain'`
+* `pointerEvents`
+  * Support setting the `pointerEvents` values via style rather than a prop.
+  * Retain the React Native specific `box-none` and `box-only` values.
+* `textShadow`
+  * Add native support for CSS text shadows.
+* `transform`
+  * Support using string values to set transforms.
+  * This can be shimmed in user-space with a parser.
+* `verticalAlign`
+  * Make `verticalAlign` the same as `textAlignVertical`.
+  * `verticalAlign === 'middle'` is the same as `textAlignVertical = 'center'`;
+* `userSelect`
+  * Support setting ability to select text via style.
+  * Equivalent to using `selectable` prop on `<Text>`.
+
+```jsx
+<View
+  style={{
+    boxShadow: '1px 1px 1px 1px #eee',
+    backgroundColor: 'hsla(50,50,50,0.5)',
+    pointerEvents: 'none',
+    transform: 'scale(0.9)',
+    width: '5rem'
+  }}
+>
+
+<Text
+  style={{
+    lineClamp: 3,
+    userSelect: 'none',
+    verticalAlign: 'middle'
+  }}
+>
+
+<Image
+  style={{
+    objectFit: 'cover'
+  }}
+>
+```
+
+### New imperative APIs
+
+* `EventTarget` API
+  * The `EventTarget` API is a implemented in JS environments, and should be available on React Native host elements. This feature is commonly used on web, and could be used by React Native developers to support more complex features without first requiring further core API changes to support each use case.
+  * Promote `node.unstable_addEventListener`, `node.unstable_removeEventListener` to stable APIs.
+  * Add `node.dispatchEvent` support.
+  * This cannot be shimmed in user-space.
+* `node.getBoundingClientRect`
+  * Synchronous element measurement API. Current APIs like `node.measure` are async and return slightly different information to what you expect from `getBoundingClientRect` on web.
+  * This cannot be shimmed in user-space.
+
+### Other
+
+* Revisit the type of all events modeled on W3C events. For example, `onTouchStart` should receive an event object that conforms to that received by the same prop in React DOM.
+* Aim to deprecate `StyleSheet.flatten()`.
```

Comment:
As long as there are missing CSS properties this will have value. I use it in project's to introspect and replace `position: 'fixed'` with `position: 'absolute'` on native platforms. Seems like a moving target that's potentially hard to keep up with.

Reply:
That introspection should not be done at runtime, as it is slow and web will not support it.